### PR TITLE
Replace schedule delay workaround with disabled entry when tests running

### PR DESCRIPTION
**Pull request by bennetelli**

### CODE REVIEW

File: src/main/java/com/redhat/cloud/policies/app/PoliciesHistoryCleaner.java [hunk 1]
```diff
@@ -28,11 +26,7 @@
      * The policies UI shows a retention time for policies history entries.
      * This scheduled job deletes from the database the history entries which are older than that retention time.
      */
-    /*
-     * TODO The scheduling is delayed to prevent an unwanted execution during tests. Remove the delay and set the period
-     * to `disabled` after the Quarkus 2 bump. See https://quarkus.io/guides/scheduler-reference for more details.
-     */
-    @Scheduled(identity = "PoliciesHistoryCleaner", delay = 10L, delayUnit = MINUTES, every = "{policies-history.cleaner.period}")
+    @Scheduled(identity = "PoliciesHistoryCleaner", every = "{policies-history.cleaner.period}")
```
Comment (gwenneg) [inline src/main/java/com/redhat/cloud/policies/app/PoliciesHistoryCleaner.java:29]:
```suggestion
    @Scheduled(identity = "PoliciesHistoryCleaner", every = "${policies-history.cleaner.period}")
```

File: src/test/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+policies-history.cleaner.period=disabled
```
Comment (gwenneg) [inline src/test/resources/application.properties:1]: This should be in the `src/main/application.properties` file with a `%test.` prefix:
```suggestion
%test.policies-history.cleaner.period=disabled
```

**Review by gwenneg — CHANGES_REQUESTED** [review]

File: src/test/resources/application.properties (hunk 2 above)
Comment (bennetelli) [reply src/test/resources/application.properties:1]: Ah, that's smarter. Thanks :)

Comment (gwenneg) [reply src/test/resources/application.properties]: Yeah both approaches work but having a single file is easier to maintain.

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**